src/__tests__/byText.test.tsx

```diff
@@ -23,9 +26,7 @@ const Banana = () => {
       <Text>Is the banana fresh?</Text>
       <Text testID="bananaFresh">not fresh</Text>
 
-      <MyButton onPress={jest.fn()} type="primary">
-        Change freshness!
-      </MyButton>
+      <MyButton onPress={jest.fn()}>Change freshness!</MyButton>
```

Comment:
Why `type="primary"` is gone here?

Reply:
The prop wasn't used, the `MyButton` component doesn't have a `type` prop, so TS would complain. I can add it there, but then it'd probably be ignored? My guess was that it'd be simpler to just remove it since it didn't seem to have any impact on the test.